store/interfaces.json

```
@@ -86,7 +86,7 @@
   "list-context.product-list": {
     "allowed": ["product-summary"],
     "composition": "children",
-    "component": "ProductSummaryList",
+    "component": "ProductSummaryListWithQuery",
```

Renaming components is a breaking change.

Even though ProductSummaryList is a good name we should keep it as being used with this component.

```suggestion
    "component": "ProductSummaryListWithQuery",
```